core/src/main/java/com/oath/oak/Chunk.java

```diff
@@ -124,34 +139,27 @@
         this.valueSerializer = valueSerializer;
     }
 
-    enum State {
-        INFANT,
-        NORMAL,
-        FROZEN,
-        RELEASED
-    }
-
     static class OpData {
         final Operation op;
         final int entryIndex;
-        final int handleIndex;
-        int prevHandleIndex;
+        final long newValueStats;
```

Comment:
As I have commented already, please change the name "stats" everywhere...

Reply:
Will do!